src/controllers/media.ts

```diff
@@ -355,6 +366,7 @@ export const getBySanitizedTitle = async(ctx: Context): Promise<MediaMetadataInt
     delete newlyCreatedResult.searchMatches;
     return ctx.body = newlyCreatedResult;
   } catch (e) {
+    console.error(e);
```

Comment:
Did we want this for the prod logs?

Reply:
I think so because we are basically hiding the error if it occurs, and it would have given us an important clue for that bug - it was encountering a duplicate key error that didn't get logged